pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -156,7 +156,7 @@ def __init__(
         self.theme = theme
 
     @property
-    def refresh_rate(self) -> int:
```

Comment:
going to block this PR for now, cc @kaushikb11 I made some changes to split out the refresh rates, need to test it though.